cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -84,11 +84,11 @@ endif()
 include(LDGOLD)
 
 include(CheckIncludeFiles)
-check_include_files("linux/memfd.h" HAVE_LINUX_MEMFD)
+check_function_exists(memfd_create HAVE_LINUX_MEMFD)
```

Comment:
This won't work IIRC, you need to supply the header and likely the library to link againts
`CMAKE_REQUIRED_INCLUDES` and `CMAKE_REQUIRED_LIBRARIES`

see: https://cmake.org/cmake/help/v3.8/module/CheckFunctionExists.html

Reply:
> see: https://cmake.org/cmake/help/v3.8/module/CheckFunctionExists.html

Cmake recommends to use CheckSymbolExists instead, sent new iteration of the patch.